ui/src/criteria/concept.tsx

```diff
@@ -121,101 +120,89 @@ function ConceptEdit(props: ConceptEditProps) {
             },
           })
           .then((res) => {
+            const data: TreeGridData = {};
+            const children: TreeGridId[] = [];
             if (res.instances) {
-              return res.instances
-                .map((instance) => {
-                  const id = instance["concept_id"]?.int64Val || 0;
-                  const row: GridRowData = {
-                    id: id,
-                    SELECTED:
-                      props.criteria.selected.findIndex(
-                        (row) => row.id === id
-                      ) > -1,
-                  };
-                  for (const k in instance) {
-                    const v = instance[k];
-                    if (!v) {
-                      row[k] = "";
-                    } else if (isValid(v.int64Val)) {
-                      row[k] = v.int64Val;
-                    } else if (isValid(v.boolVal)) {
-                      row[k] = v.boolVal;
-                    } else {
-                      row[k] = v.stringVal;
-                    }
+              // TODO(tjennison): Use server side limits.
+              res.instances.slice(0, 100).forEach((instance) => {
+                const id = instance["concept_id"]?.int64Val || 0;
+                if (id === 0) {
+                  return;
+                }
+
+                const row: TreeGridRowData = {
+                  view_hierarchy: (
+                    <IconButton size="small">
+                      <AccountTreeIcon fontSize="inherit" />
+                    </IconButton>
+                  ),
+                };
+                for (const k in instance) {
+                  const v = instance[k];
+                  if (!v) {
+                    row[k] = "";
+                  } else if (isValid(v.int64Val)) {
+                    row[k] = v.int64Val;
+                  } else if (isValid(v.boolVal)) {
+                    row[k] = v.boolVal;
+                  } else {
+                    row[k] = v.stringVal;
                   }
-                  return row;
-                })
-                .filter((row) => row.id !== 0);
+                }
+
+                row["standard_concept"] = !row["standard_concept"]
```

Comment:
nit: stared at this for a while before correctly parsing; I think just `row["standard_concept"]` as the predicate is probably a bit clearer. I also think moving it into the above for loop might be slightly clearer to avoid the in-place modification.

Reply:
Done.